gaming/README.md

```diff
@@ -0,0 +1,153 @@
+# Spanner Gaming Samples
+
+This repository contains sample code for the following use-cases when using Cloud Spanner for the backend:
+
+- Player creation, login, and skin changes
+
+## How to use this
+
+### Setup infrastructure
+A terraform file is provided that creates the appropriate resources for these samples.
```

Comment:
(Similar comment as usage of Wrench below). Using terraform to manage resources distracts the reader from the core goals (at least my understanding) of sample apps. While terraform may be a good topic to dive into, whether this should be included in sample apps at the expense of diluting other goals is a separate conversation. Additionally, we already have codelabs for terraform -https://codelabs.developers.google.com/codelabs/cloud-spanner-terraform#2. 

I'd recommend not using terraform in this PR. Consider following up with a separate PR after thorough evaluation with team members.